src/osd/mClockOpClassQueue.cc

```diff
@@ -106,10 +106,6 @@ namespace ceph {
 
     if (osd_op_type_t::client_op != type) {
       return type;
-    } else if (MSG_OSD_SUBOP ==
```

Comment:
@liewegas In a previous version of this commit you said:

> @ivancich The subops are being removed (they were only for jewel compatibility). I have a feeling this code really supposed to be matching MOSDRepOp[Reply?] instead?

This was done under the guidance of Sam, and I'm pretty certain subops were mentioned in overviews and reviews, and I don't recall Sam ever flagging the term. So I'm thinking subops are what he intended. @jdurgin do you have any recollection of discussing this with Sam?

That said, I can easily separate out rep ops and rep op replies, so they can be configured separately. That'd be useful in tuning replication relative to other operations.

Reply:
He probably said subops out of habit.. that is the old message type.  They were replaced with repops in the hammer or jewel timeframe (i forget).  In any case, s/subop/repop/ is probbaly what you want if it is just replicated pool ops you care about.  If you also want to include ec osd to osd operations, there are a bunch of other message types to match.  This is probably a bit list of message type ids to match against.

Reply:
For now I will comment it out...